### ISSUE TITLE
Some details about the project

### ISSUE THREAD

**Issue by QQQhz** [IvLabs/stagewise-knowledge-distillation#15]
I would like to ask you how the teacher model of Imagenette (BS =64) in this project was trained.Because I went to the training by myself, the effect was very poor, so I would like to ask you this question, thank you

**Comment by akshaykulkarni07** [comment]
Hi, please note that the Imagenette dataset changed their train-val split on Dec 6, 2019 ([mentioned here](https://github.com/fastai/imagenette)) by which time we had already completed much of our experimentation with the older train-val split. In the older train-val split, the validation set was smaller which gave better accuracy numbers. This is one possible reason for you getting lower accuracy since you may have used the newer train-val split.

For our paper, we consistently used the older train-val split in all experiments and thus our comparisons remain valid though they won't be comparable to the new train-val split.

**Comment by QQQhz** [comment]
> Hi, please note that the Imagenette dataset changed their train-val split on Dec 6, 2019 ([mentioned here](https://github.com/fastai/imagenette)) by which time we had already completed much of our experimentation with the older train-val split. In the older train-val split, the validation set was smaller which gave better accuracy numbers. This is one possible reason for you getting lower accuracy since you may have used the newer train-val split.
> 
> For our paper, we consistently used the older train-val split in all experiments and thus our comparisons remain valid though they won't be comparable to the new train-val split.

Thank you for your answer.I just used the old version of Imagenette, but this may be because of my lack of experience. For the first time, I studied a relatively large data set, so the setting of super parameters for the training process was not very good, so the accuracy was extremely low.Is it convenient for you to send me your method or relevant code for training the teacher model of Imagenette?Thank you very much, because I've been stuck in this place for a long time!
My email: hfooru@gmail.com

**Comment by akshaykulkarni07** [comment]
We used this [Jupyter notebook](https://github.com/IvLabs/stagewise-knowledge-distillation/blob/master/image_classification/notebooks/resnet_pretraining.ipynb) for training the teacher models. This linked one loads CIFAR10 at the start, but the rest of the training code was same for all the datasets we used.

**Comment by QQQhz** [comment]
Thank you very much. I'll have a try
